### DISCUSSION TITLE
Is any prometheus metrics available for Kiali and Kiali operator ?

### DISCUSSION THREAD

**Discussion by Rachitmehrotra** [kiali/kiali#5379] [General]

**Comment by Rachitmehrotra** [comment]
Hello Team,
Do we have any promethues metrics for Kiali and Kiali operator; if yes please point me correct link.

**Comment by Xunzhuo** [comment]
Yes and see: https://kiali.io/docs/installation/deployment-options/#metrics-server

**Reply by Rachitmehrotra** [reply]
Hello @Xunzhuo 
For Kiali operator is thr any link for metrics

**Reply by jmazzitelli** [reply]
operator: https://github.com/kiali/helm-charts/blob/master/kiali-operator/values.yaml#L25-L27
they are over the 8080 port: https://github.com/kiali/helm-charts/blob/master/kiali-operator/templates/deployment.yaml#L97-L99

**Reply by jmazzitelli** [reply]
(the operator metrics are not extensive, and they come from the operator SDK itself (the kiali operator doesn't emit its own custom metrics - the metrics you get are the common metrics that the Operator SDK emit).
